### ISSUE TITLE
Keep workspace icon instead of name with waybar?

### ISSUE THREAD

**Issue by tmpm697** [pierrechevalier83/workstyle#39]
I have waybar setting like:
```
format-icons": {
  "10":  "<This is an icon>"
```
That will display only an icon for workspace name. 

If I run `workstyle`, it will convert icon to workspace that mapped from sway/config, how to have icons as workspace and also display applications next to it?

also asked here: https://github.com/Alexays/Waybar/issues/1521#issue-1206215273

**Comment by jkim5959** [comment]
yeah i second this. I actually wanna get rid of workspace number and just use application icons since the colouring will distinguish different workspaces anyway.
